Tests/test_imagemath_lambda_eval.py

```diff
@@ -176,8 +176,9 @@ def test_compare() -> None:
         )
         == "I 2"
     )
-    assert pixel(ImageMath.lambda_eval(lambda args: args["A"] == 1, images)) == "I 1"
-    assert pixel(ImageMath.lambda_eval(lambda args: args["A"] == 2, images)) == "I 0"
+    for i in range(1, 3):
+        assert ImageMath.lambda_eval(lambda args: args["A"] == i, images) is False
```

Comment:
Not related to this changes, but for me this syntax should be deprecated in favor to more pythonic:
```suggestion
        assert ImageMath.lambda_eval(lambda args: args["A"] == i, **images) is False
```

Reply:
I've pushed a commit to update how it is called in the tests.

I'm guessing you're not suggesting we actually deprecate any functionality, as we're not doing anything special with `kw`, so this would just be natural Python handling of keywords.

Reply:
Actually I am :-)

For me there are no reasons to support two ways for passing arguments. I see that Python's `eval` gets dicts as arguments, so we could adopt only this and remove kwargs, or keep kwargs only since this is more convenient and pythonistic.

Reply:
Oh, I see now, you want to deprecate `options` or `**kw` - https://github.com/python-pillow/Pillow/blob/d05262105ada44e79f10736823cef7105de795c1/src/PIL/ImageMath.py#L241-L261

Reply:
I've removed these changes from here, and created https://github.com/python-pillow/Pillow/pull/8242 instead.